api/models/schema.py

```diff
@@ -297,6 +300,12 @@ class Place(SchemaBaseModel):
                     "or area coverage extent."
     )
 
+    @validator('geo', 'name')
```

Comment:
Aren't these required properties already validated as such since they are marked as required?

Reply:
Both these attributes for the Place class are optional. This validation makes sure that we have value for at least one of those attributes. Otherwise, we will end up with a Place with value for type attribute only.

Reply:
Then we must be able to express these validations constraints using JsonForms because that's it's purpose. Let's discuss tomorrow and try to come up with a clean solution.

Reply:
Isn't the pydantic model(s) is the Json Schema? If you are referring to the schema used in UI to make sure one of these attributes has a value, I think we still need at pydantic model level as the endpoints can be called by other clients.

Reply:
Yes, but it was my understanding that these schema validations (`required` and `optional`) where already picked up by the API. @sblack-usu, please confirm.

In any case, these validations need to be applied in a way that they get reflected in our JsonSchema because our forms need to provide immediate error feedback while the form it's being filled.

Reply:
Validators are not translated to a json schema.  If we used the validator, the validation would not be present in the generated form.

Reply:
I believe there is another way to express this use case that will be present in the json schema that generates.  I will look into it.

Reply:
I think this solution is fine, but perhaps use a root validator instead to help with the error messaging (The current implementation would show an error for the name attribute.  There is a way to express this in the json schema... but I don't know if jsonforms supports that expression.  More information at https://github.com/pydantic/pydantic/issues/506

I think for an edge case like this, we can skip the jsonforms validation and just provide api validation.

Reply:
Agree with @sblack-usu. I've looked at ways to express this type of conditional requiring in JsonSchema before and our renderers would not support it.